docs/source/reference/config.rst

```diff
@@ -193,6 +193,32 @@ Available fields and semantics:
     # Default: false.
     disk_encrypted: false
 
+    # Reserved capacity (optional).
+    #
+    # Whether to prioritize capacity reservations (considered as 0 cost) in the
+    # optimizer.
+    #
+    # If you have capacity reservations in your AWS project:
+    # Setting this to true guarantees the optimizer will pick any matching
+    # reservation within all regions and AWS will auto consume your reservations
+    # with instance match criteria to "open", and setting to false means
+    # optimizer uses regular, non-zero pricing in optimization (if by chance any
+    # matching reservation exists, AWS will still consume the reservation).
+    #
+    # Default: false.
+    prioritize_reservations: false
```

Comment:
Why should one set this to false..? Doesn't the reservation indeed cost nothing after its first time purchase?

Reply:
It is because `prioritize_reservations` will lead to significant additional time for optimization, as we have to check the available reservations in each region, which can be very slow. We turned it off for a better performance for most non reservation users.

Reply:
Good point! If this is the case, adding this to the doc might help address some confusion it may raise

Reply:
Add some additional lines for this.